git clone onto a case-insensitive file system is problematic

There are name collisions that confuse git ( VERSION and version/ for example ). Active development will happen on Linux of course, but for casual clones to browse source on MacOS this can be confusing since it shows files as deleted immediately after a clone.

Even after changing the file from VERSION to VERSION.txt this won't work because overlay libs won't compile on any arch other than linux (as expected):

```
><> make
+ clean
rm -f img
rm -f -r /Users/bacongobbler/go/src/github.com/jessfraz/img/cross
+ img
go build -tags "" -ldflags "-w -X github.com/jessfraz/img/version.GITCOMMIT=b6a9fcf-dirty -X github.com/jessfraz/img/version.VERSION=v0.2.4" -o img .
worker/runc/workeropt.go:15:2: build constraints exclude all Go files in /Users/bacongobbler/go/src/github.com/jessfraz/img/vendor/github.com/containerd/containerd/snapshots/overlay
make: *** [img] Error 1

><> head -n 1 vendor/github.com/containerd/containerd/snapshots/overlay/overlay.go
// +build linux
```

So yeah this project will compile for you only if you're a penguin

--- a/README.md
+++ b/README.md
@@ -35,6 +35,8 @@ For the FUSE backend, you will also need `fusermount` installed.
 $ go get github.com/jessfraz/img
 ```
 
+NOTE: Only works on Linux. Compile and run in a container (explained below) if you're on Windows or MacOS ;)
+
 #### Running with Docker
 
 ```console

